### PR TITLE
WIP: automatically update pronouns in profile

### PR DESCRIPTION
**Pull request by lukec11**
This is a work in progress! Should not be merged yet.

### CODE REVIEW

**Comment by sampoder** [comment]
Got distracted, tested a little now

File: app.js [hunk 1]
```diff
@@ -542,6 +564,15 @@ async function startTutorial(user, restart) {
   }
 }
 
+async function setPronounsOnProfile(userId, pronouns) {
```
Comment (sampoder) [inline app.js:567]: can confirm this function works in changing profile fields.

**Comment by sampoder** [comment]
@MatthewStanciu I tested the Pronouns part, it should work (you can test it in the Test workspace, just it will change your display name as an alternative). Feel free to go over the rest

**Comment by MatthewStanciu** [comment]
Looks great! Love the channels update as well. Merging now

**Pull request merged**